### PR TITLE
Cfkawa patch 1

### PR DESCRIPTION
**Pull request by cfkawa**
Created function to load data for LSTM. Splits data into test and train

## Purpose
Created function to load data for LSTM

## Type of change
New Feature (LSTM function)
_What types of change is it?_
_Select the appropriate type(s) that describe this PR_

- Bugfix (non-breaking change which fixes an issue)
- New feature (non-breaking change which adds functionality)
- Breaking change (non-backwards-compatible fix or feature)
- Code style update (formatting, renaming)
- Refactoring (no functional changes, no API changes)
- Repository chores (changes made to improve repository/workflow quality)
- Documentation update
- Other (please describe)

## Checklist
_Put an `x` in the boxes that apply._

- [x ] My code functions as intended
- [x ] I have used camelCase in my variable and function names
- [] I have commented my code properly
- [ ] I have provided the necessary prompts for automatic documentation
- [ ] I have provided a toy case under the examples folder
- [ ] I have addressed all the issues entirely

### CODE REVIEW

File: LSTM/Load_data.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+import seaborn as sns
+import tensorflow as tf
+from sklearn.model_selection import train_test_split
+from os.path import exists as file_exists
+from datetime import datetime
+
+
+
+def loadData(image_path,label_path): 
+    xData=np.load(image_path)
```
Comment (sahilbhola14) [inline LSTM/Load_data.py:19]: Use camelCase for *image_path*

File: LSTM/Load_data.py [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+import pandas as pd
+import seaborn as sns
+import tensorflow as tf
+from sklearn.model_selection import train_test_split
+from os.path import exists as file_exists
+from datetime import datetime
+
+
+
+def loadData(image_path,label_path):
```
Comment (sahilbhola14) [inline LSTM/Load_data.py:18]: Add docstrings for documentation

**Review by sahilbhola14 — CHANGES_REQUESTED** [review]

**Comment by sahilbhola14** [comment]
Run pre-commit. See [README.md](https://github.com/PinguDevTeam/ICF-MI-framework/blob/main/README.md) for instructions

**Comment by sahilbhola14** [comment]
Please use sensible names for PR. `Cfkawa patch 1` may make sense to you but is not informative to the rest. Naming is important for debugging purpose.

**Comment by dorukaks** [comment]
Please refer to the guidelines in [formats](https://github.com/PinguDevTeam/ICF-MI-framework/tree/main/formats) directory for commit message and PR style.

**Pull request closed without merging**